shared/scss/views/_privacy-practices.scss

```diff
@@ -0,0 +1,125 @@
+.privacy-practices {
+    min-height: 100%;
+    box-sizing: border-box;
+}
+
+.privacy-practices__overview,
+.privacy-practices__explainer,
+.privacy-practices__attrib {
+    text-align: center;
+}
+
+// the domain duplicate the same styles
+// as the ones in the main view, but they aren't reusable
+// because of some overriding rules for the subviews
+// TODO DRY this up
+.privacy-practices__overview__domain {
+    line-height: 1.4em;
+    height: 1.2em;
+    display: block;
+    max-width: $popup__width - 125;
+    font-weight: bold;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    margin-top: 7.5px;
+    text-align: center;
+    width: 100%;
+    max-width: 100%;
+    font-size: 17px;
+}
+
+.privacy-practices__overview__subtitle {
+    padding-top: 7.5px;
+    @include uppercase_label();
+}
+
+.privacy-practices__overview__icon {
+    width: 48px;
+    height: 65px;
+    margin: 0 auto;
+    background-repeat: no-repeat;
+}
+
+// TODO dry these up also
+.privacy-practices__overview__icon--good {
+    .is-browser--chrome & {
+        background-image: url($icon__hero_ribbon__good--chrome);
```

Comment:
Actually this could be improved even further. I see a lot of these .is-browser--chrome and .is-browser--firefox around, we should probably have a mixin for browser-specific styles...
In addition to that, we could put the base extension URLs in a variable, so instead of having 30 variations of $icon-1--chrome and $icon-1--firefox, we'd just pass along the icon name and we'd have something like:
<pre>
@mixin browser_icons_path($icon-name) {
     .is-browser--moz & { 
       background-image: url($moz-url$icon-name);
    }
    .is-browser--chrome & { 
       background-image: url($chrome-url$icon-name);
    }
}
</pre>

Just an idea, and not critical ofc, but I think it'd save us time with the other layouts